src/hooks/useTransfer.js

```diff
@@ -41,10 +48,27 @@ export const useTransferToL2 = () => {
         const isEthToken = utils.token.isEth(symbol);
         const bridgeContract = getTokenBridgeContract(bridgeAddress);
         const depositHandler = isEthToken ? depositEth : deposit;
+        const tokenContract = isEthToken && getTokenContract(tokenAddress);
```

Comment:
why the && ?

Reply:
this line was moved up from line 47. this variable is relevant only if **isEthToken** is true, and this is the shortest way to write it.

Reply:
Why do u need this statement?